packages/vue3/package.json

```diff
@@ -71,7 +71,11 @@
     "parserOptions": {
       "ecmaVersion": 2020
     },
-    "rules": {},
+    "rules": {
+      "no-empty-function": "off",
```

Comment:
does it affect only tests or the src code as well?

Reply:
removed